packages/components/src/components/list-box/_list-box.scss

```diff
@@ -611,9 +611,9 @@ $list-box-menu-width: rem(300px);
 
   // Tweaks for descendants
   // When handling input, we need to target nodes that specifically opt-in to
-  // the `combobox` role in order to make sure the text input is styled
+  // the type text in order to make sure the text input is styled
   // correctly.
-  .#{$prefix}--list-box input[role='combobox'] {
+  .#{$prefix}--list-box input[type='text'] {
     background-color: inherit;
     min-width: 0;
   }
```

Comment:
Focus styles on multi select need to be added
<img width="360" alt="Screen Shot 2020-04-02 at 12 41 47 PM" src="https://user-images.githubusercontent.com/11928039/78294756-c0d5fd80-74df-11ea-8903-03c546256c00.png">